### ISSUE TITLE
Object streaming based on multi-sourcing 

### ISSUE THREAD

**Issue by gedw99** [nats-io/nats-server#3197]
Multi sourcing is a term from video streaming and it is when a client streams content from many pieces.

IPFS also has this feature.

It’s useful because you can get more throughout on the client and your effectively load balancing 

now nats.go can do load balancing and so can nats.js 

So it should be possible for a web client to stream chunks from many sources of nats object streams ?

Does it already do this when I use nats.go client with a cluster of nats Jetstream servers ?

**Comment by derekcollison** [comment]
Not sure object store would want to do this but have thought about it for another problem space.

Also these streams (all including KV and Object) can have mirrors, digital twins per se that may be closer or have more bandwidth or less use. When you get StreamInfo we list these as alternates sorted by RTT from the requestor.

**Comment by gedw99** [comment]
Yes it def applies to lots of use cases, not just video streaming. 

streamInfo looks like it will help me. Thankyou !

will close this and reopen if needed.
